adapter/integration-test/src/adapter-datasource.test.js

```diff
@@ -98,6 +98,18 @@ describe('Datasource Configuration', () => {
     expect(datasourceResponse.status).toEqual(400)
   }, TIMEOUT)
 
+  test('Should create datasource with long URL [POST /datasources]', async () => {
+    const longUrlDatasourceConfig = getDatasourceConfig()
+    const queryParameter = '&parameter=longParameter'
+    const longUrl = 'http://www.disresprect.com?first=test' + queryParameter.repeat(20)
```

Comment:
Please use other URL (seems not appropriate)

Reply:
done.

there are other occurrences of domains like that in the test case (not written by me), should i change those as well?

Reply:
yep, please use the scouts rule => leave place a little cleaner than you found it :)